dbschema/progress-report.esdl

```diff
@@ -8,6 +8,8 @@ module default {
     workflowEvents := .<report[is ProgressReport::WorkflowEvent];
 
     single varianceExplanation := .<report[is ProgressReport::VarianceExplanation];
+    
+    summaries := .<report[is ProgressSummary];
```

Comment:
I can't see this being that helpful. I don't think app code will use it. Nor will adhoc exploration benefit.

Adhoc exploration would probably be looking for the cumulative summary.
I'd be ok with that
```edgeql
single summaryCumulative := (select .<report[is ProgressSummary] filter .period = ProgressSummary::Period.Cumulative)
```

Reply:
I thought that you wanted me to backfill manually based on our conversation on slack yesterday. I am ok removing it.

Reply:
Sorry. I was just answering on the "automatic backfill" question. Works for me. We can always add/change later too.